backend/src/data/repositories/users-to-groups/users-to-groups.repository.ts

```diff
@@ -32,6 +32,17 @@ class UsersToGroups {
     return groups ?? null;
   }
 
+  public async getUsersByGroupId(
+    groupId: number,
+  ): Promise<UsersToGroupsM[] | null> {
+    const users = await this.#UsersToGroupsModel
+      .query()
+      .select()
+      .where({ groupId });
+
+    return users ?? null;
```

Comment:
```suggestion
  ): Promise<UsersToGroupsM[]> {
    const users = await this.#UsersToGroupsModel
      .query()
      .select()
      .where({ groupId });

    return users;
```
it will an empty array by default